### PR TITLE
Updating output attribute in place order mutation output object

### PR DESCRIPTION
**Pull request by VinothKumar361**
## Purpose of this pull request

This pull request (PR) updates order_id which is deprecated in outupt object of place order mutation in graphql

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/mutations/place-order.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/a3c2af0498d21e3f689bc8dd690d7c633a025276/app/code/Magento/QuoteGraphQl/etc/schema.graphqls#L367

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/graphql/mutations/place-order.md [hunk 1]
```diff
@@ -73,8 +73,8 @@ Attribute |  Data Type | Description
 
 Attribute |  Data Type | Description
 --- | --- | ---
-`order_id` String | The unique ID that identifies the order
-`order_number` String | The unique ID that identifies the order
+`order_id` |String! | Deprecated.Use order_number instead.The unique ID that identifies the order
```
Comment (rogyar) [inline src/guides/v2.3/graphql/mutations/place-order.md]: Hi @VinothKumar361. Please, make sure that the number of whitespaces is proper for both lines

```suggestion
`order_id` | String! | Deprecated. Use order_number instead. The unique ID that identifies the order
```

File: src/guides/v2.3/graphql/mutations/place-order.md [hunk 2]
```diff
@@ -73,8 +73,8 @@ Attribute |  Data Type | Description
...
+`order_number`| String | The unique ID that identifies the order
```
Comment (rogyar) [inline src/guides/v2.3/graphql/mutations/place-order.md]:
```suggestion
`order_number` | String | The unique ID that identifies the order
```

**Comment by VinothKumar361** [comment]
Hi @rogyar , changes has been done.

**Pull request merged**